src/ui/dfn-list.js

```diff
@@ -49,9 +49,33 @@ function show() {
           <a href="${`#${dfn.id}`}">
             ${dfn.textContent}
           </a>
+          ${labelDfnIfExported(dfn)} ${labelDfnIfUnused(dfn)}
         </li>
       `;
     });
   render`${definitionLinks}`;
   ui.freshModal(l10n.list_of_definitions, ul, button);
 }
+
+/**
+ * If a definition is exported, label it accordingly
+ * @param {HTMLElement} dfn a definition
+ */
+function labelDfnIfExported(dfn) {
+  if (dfn.hasAttribute("data-export")) {
+    return hyperHTML`<span class="exported">(exported)</span>`;
+  }
+  return null;
+}
+
+/**
+ * If a definition is unused, label it accordingly
+ * @param {HTMLElement} dfn a definition
+ */
+function labelDfnIfUnused(dfn) {
+  const isUsed = document.querySelector(`a[href^="#${dfn.id}"]`);
```

Comment:
Nit: maybe use this for consistency with labelDfnIfExported
```js
  const isUsed = document.querySelector(`a[href^="#${dfn.id}"]`);
  if (!isUsed) {
    return hyperHTML`<span class="unused">(unused)</span>`;
  }
  return null;
```

Reply:
hm. one `!` so i'm still okay with it. XD

if we're going with that, then for overall consistency, we should change `labelDfnIfExported/1` to
```javascript
function labelDfnIfExported(dfn) {
  const isExported =  dfn.hasAttribute("data-export");
  if (isExported) {
    return hyperHTML`<span class="exported">(exported)</span>`;
  }
  return null;
}
```